### ISSUE TITLE
No values found for wildcard

### ISSUE THREAD

**Issue by pupdike** [adieyal/sd-dynamic-prompts#245]
I am using v2.5.6 of Dynamic Prompts and I am using the latest automatic1111 in Windows 11

I have the checkbox checked for "Dynamic Prompts enabled" after installing it

If I type {dog|football} I successfully get just one of these (and not both) so the extension seems to be installed.

But if I try any wildcard I get the following message in the console: 
No values found for wildcard folder/name

The Wildcards manager tab sees all my wilcards, and I can copy the path from there
__folder/name__

So for example if I try this prompt: __jumbo/items/materials/decorative_stones__

It will not be interpreted as one of the items inside the file and I get the same error message:
No values found for wildcard jumbo/items/materials/decorative_stones

The extension still works in my older build of automatic1111

Note that I am not running any other extensions currently while trying to debug this.  I tried to find that error message in the source by searching all the files using Notepad++ and I can't seem to figure out where it is coming from so its hard to track down the issue.

If anybody knows where the error message is generated I can probably debug myself.

Thanks!

**Comment by adieyal** [comment]
Could you double-check that you have a folder called wildcards inside of the extensions/sd-dynamic-prompt folder? And then check that extensions/sd-dynamic-prompt/wildcards/jumbo/items/materials/decorative_stones.txt exists?

**Comment by pupdike** [comment]
I checked and it was there.  But I did some further exploration and I think I know the source of the problem.  I am running multiple versions of automatic1111, a stable older version and the latest.  In order to avoid forking my wildcards I use directory junctions to point to a single shared copy of them.  In my older version of automatic1111 sd-dynamic-prompts can use a directory junction just fine for the wildcards folder.  But that doesn't seem to work with the latest automatic1111.  Its weird that the Wildcards Manager tab can use the junction just fine, but it generates the "No value found for wildcard" error when actually using a wildcard in a prompt. So my workaround is to duplicate my wildcards folder.  This will introduce some technical debt but at least I have wildcards working again.  I suppose I can just merge my wildcards later.  Thanks!
